test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -237,6 +237,18 @@ func testSecondaryNetwork(t *testing.T, networkType string, pods []*testPodInfo)
 			t.Fatalf("Error when pinging between interfaces: %v", err)
 		}
 	})
+	t.Run("testRetstartAntreaAgentPods", func(t *testing.T) {
+		if err := e2eTestData.RestartAntreaAgentPods(defaultTimeout); err != nil {
```

Comment:
Could we remove some Pods after agent stops and check stale OVS ports are removed?

Reply:
sure